slicemachine.config.json

```diff
@@ -1,6 +1,7 @@
 {
   "libraries": ["./src/slices"],
   "adapter": "@slicemachine/adapter-next",
-  "repositoryName": "nextjs-starter-prismic-multi-language",
+  "repositoryName": "i18n-lucie",
```

Comment:
I assume this is just a temporary switch, yeah? That after everything is deployed we'll update the `nextjs-starter-prismic-multi-language` repo and switch back to it?

Reply:
Exactly yes :)